examples/src/examples/graphics/render-cubemap.tsx

```diff
@@ -249,34 +220,32 @@
             camera.lookAt(pc.Vec3.ZERO);
 
             // project textures, and display them on the screen
-            var srcCube = shinyBall.script.cubemapRenderer.cubeMap;
+            // @ts-ignore engine-tsd
+            const srcCube = shinyBall.script.cubemapRenderer.cubeMap;
 
             // cube -> equi1
-            pc.reprojectTexture(srcCube, textureEqui, {
-                numSamples: 1
-            });
+            pc.reprojectTexture(device, srcCube, textureEqui, undefined, 1);
```

Comment:
have you integrated master to this branch in last few weeks? I think the syntax on left with options parameters is the last one, not passing params as here. Worth checking.

Reply:
Oh yeah, i'll bump the version of pc to 1.42.0

Reply:
ah that's how it works. Will it work fine to bump it to a version we're about to release to release examples at the same time? Or will we release new version of the engine, then bump up version here and then release engine examples?